wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/Devices/ProfileDeviceDetailsView.swift

```diff
@@ -101,12 +110,12 @@ struct ProfileDeviceDetailsView: View {
     var body: some View {
         NavigationView {
             ScrollView {
-                if let thumbprint = viewModel.mlsThumbprint, thumbprint.isNonEmpty {
+                if viewModel.isE2eIdentityEnabled, let thumbprint = viewModel.mlsThumbprint, thumbprint.isNonEmpty {
```

Comment:
**sugesstion:**  For not activated case, mlsview should not be shown as we no longer have mls thumbprint

Reply:
@theswiftway is this what you mean?
f1b693e0c89441ef9311f3c53e71f080a06e8c9c